spec/models/plant_accession_spec.rb

```diff
@@ -47,6 +47,25 @@
     end
   end
 
+  context 'unique attributes' do
+    it 'disallows duplicate plant_accession and originating_organisation' do
```

Comment:
Index definition can also be tested with [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_record/have_db_index_matcher.rb).